app/components/post_list/post/post.tsx

```diff
@@ -223,7 +225,7 @@ const Post = ({
         }
 
         Keyboard.dismiss();
-        const passProps = {sourceScreen: location, post, showAddReaction, serverUrl};
+        const passProps = {sourceScreen: location, post, showAddReaction, serverUrl, files};
```

Comment:
Why do we need the files so far back? Why not adding the observer in the post options screen or even in the edit screen?